edit/linter-report.js

```diff
@@ -0,0 +1,161 @@
+/* global linter editors clipString createLinterHelpDialog makeSectionVisible */
+'use strict';
+
+var linterReport = (() => { // eslint-disable-line no-var
```

Comment:
I think `linterReport` is better. When I see `linter.report`, I would expect that `report` is a module member exported by `linter.js`.

Reply:
hmm indeed... but I still want to avoid an additional global variable for something that only makes sense as a part of `linter` housekeeping. It's pretty common to add things to an existing object in a different file so this is not a problem. Maybe there's no need to expose the report at all, just expose a few functions on the `linter` object?